webpack.config.client.js

```diff
@@ -56,6 +56,13 @@ module.exports = ({
       },
       disableHostCheck: true,
     },
+    resolve: {
+      fallback: {
+        // Override webpacks default handling for these as they arnt availible on the client.
+        fs: false,
```

Comment:
From the [Clean up configuration](https://webpack.js.org/migrate/5/#clean-up-configuration) section of the webpack upgrade docs

> If you are using something like node.fs: 'empty' replace it with resolve.fallback.fs: false.